src/axom/slam/policies/SubsettingPolicies.hpp

```diff
@@ -44,7 +44,7 @@ namespace policies
 
 struct NoSubset
```

Comment:
for these cases, should you export the struct that conatins these memberrs?

Reply:
I'll try to add `AXOM_EXPORT` here instead of the static member and report back.

(Note that the current solution builds and passes all tests in our uberenv/vcpkg builds)

Reply:
@cyrush -- your suggestion also removes the link error, but introduces lots of link warnings of the form:
```
LINK : warning LNK4286: symbol '??0NoSubset@policies@slam@axom@@QEAA@XZ (public: __cdecl axom::slam::policies::NoSubset::NoSubset(void))' defined in 'OrderedSet.obj' is imported by 'State.obj' [<buid-axom>\axom\slam\examples\tinyHydro\slam_tinyHydro_sedov_ex.vcxproj]
```
This is the same thing we saw in #548.

On the other hand, marking the static members (as in the code that is currently checked in), does not appear to have these linker warnings (see the build logs for https://github.com/LLNL/axom/actions/runs/924399092)

I think we should keep the current solution (tagging the static members rather than the class/struct).

Reply:
Ok, that's a bit strange.

I suspect it has to do with the fact that these are header only and static members are going to be tricky b/c there isn't explicit storage associated with one compilation unit.  This may be a signal that future mysteries await us. 

Regardless, I agree with keeping what you have now.